docs/manual/kernel/Introduction.tex

```diff
@@ -108,8 +108,8 @@ \subsection{Multi-Core Systems}
 \subsubsection{Example Code}
 To use the mailbox mechanism all that's required a pointer to the function to run on the remote core. In the example we'll assume core 5 exists and we want to run \verb|ExampleFunction()| on that core. The callback function can optionally have an argument passed to it at runtime, but it's not required so we'll use \verb|nullptr|. We omit the argument in this example. The function we'll use is defined in \verb|interrupts/Ipi.h|.
 
-\begin{verbatim}
+\begin{lstlisting}
```

Comment:
not an issue here, but the rendering of \begin has changed. Just double check that is not an issue with some weird non-printable characters.